src/app/routes/stake/stake-list/stake-list-card/stake-list-card.component.scss

```diff
@@ -0,0 +1,116 @@
+::ng-deep .mat-card-header-text {
+    margin: 0px !important;
+    display: none; 
+}
+
+*{
```

Comment:
I thought this is set by default. @davycastillo this font should be the default one or just in few views?

Reply:
It wasn't. The text was using Roboto initially, that is why I imported it locally and using it like that. Do I make it a global import?

Reply:
I see, we have this import in index.html. If that isn't working then please, update it to be a global import. Thanks 👍